client/my-sites/checklist/wpcom-checklist/wpcom-task-list.js

```diff
@@ -31,7 +31,8 @@ function getTasks( {
 	const tasks = [];
 	const segmentSlug = getSiteTypePropertyValue( 'id', siteSegment, 'slug' );
 
-	const getTask = taskId => get( taskStatuses, taskId );
+	const getTask = taskId =>
+		taskStatuses ? taskStatuses.filter( task => task.id === taskId )[ 0 ] : undefined;
```

Comment:
Can we rely on `taskStatuses` being an array, and therefore have a `filter()` method on its prototype?

I didn't notice any errors on the frontend.

Reply:
That's guaranteed by the reducer

Reply:
Also ideally we'll delete the "old checklist" code soon :-)